src/Spago/Bower.hs

```diff
@@ -93,7 +93,7 @@ mkPackageName spagoName = do
 mkBowerVersion :: Spago m => Bower.PackageName -> Text -> Text -> m Bower.VersionRange
 mkBowerVersion packageName version repo = do
 
-  let cmd = "bower info --json '" <> Bower.runPackageName packageName <> "#" <> version <> "'"
+  let cmd = "bower info --json " <> Bower.runPackageName packageName <> "#" <> version
```

Comment:
Thanks @f-f 

The ticks were to try and reduce the chances of versions (particular branch/tag names) with spaces in causing problems.

However, just adding ticks is not a very secure solution. I will see if `procStrictWithErr` will work instead.

Reply:
Would it work to use double quotes?

Reply:
It would work I guess and probably by fine... but I think constructing shell strings like this tends to result in bugs (e.g. what if a branch name has a quote in it) and security holes.

I guess I could finish this issue using `shell` and double quotes and then look for a better solution afterwards.